app/src/main/java/org/schabi/newpipe/util/DeviceUtils.java

```diff
@@ -103,7 +106,6 @@ public static int spToPx(@Dimension(unit = Dimension.SP) final int sp,
      */
     public static boolean shouldSupportMediaTunneling() {
         return Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
-                && !HI3798MV200;
+                && !HI3798MV200 && !CVT_MT5886_EU_1G;
```

Comment:
Should I do the following instead?
```suggestion
                && !HI3798MV200
                && !CVT_MT5886_EU_1G;
```

Reply:
Yep, below each other is better readable :)